kornia/__init__.py

```diff
@@ -6,7 +6,7 @@
     raise RuntimeError("Kornia requires Python 3.6.0 or later")
 
 try:
-    from .version import __version__  # noqa: F401
+    from .version import __version__
```

Comment:
@cclauss not related to this PR but do you have any suggestion or good practices in the way we generate the versioning ?

Reply:
I kinda view linting suggestions as _strong recommendations_ or _warnings_ but not at the same level as syntax errors.  I therefore think that an occasional _linter directive_ (like `# noqa: F401`) is akin to saying ___I am over 18 and I know what I am doing___.  I believe that an occasional linter directive is quite acceptable (especially when it is near the offending code, and not hidden is some `.flake8` file).  I dislike pylint for two reasons...  It is not very good at finding _undefined names_ (flake8 F821) and it forces too many linter directives.  So if this code seems like a good way to get the version then go ahead and use a linter directive.  It ain't a crime to occasionally tel the linter to shut up.